SPECS/azurelinux-image-tools/azurelinux-image-tools.spec

```diff
@@ -93,12 +93,17 @@ go test -C toolkit/tools ./...
 %license LICENSE
 %{_bindir}/imagecustomizer
 # Container support files - placed in container filesystem paths with imagecustomizer- prefix
-/usr/local/bin/imagecustomizer-entrypoint.sh
-/usr/local/bin/imagecustomizer-run.sh
-/usr/local/bin/telemetry_hopper.py
-/imagecustomizer-telemetry-requirements.txt
+%{_prefix}/local/bin/imagecustomizer-entrypoint.sh
```

Comment:
These shouldn't be in `local`.

Reply:
I was following here - https://github.com/microsoft/azure-linux-image-tools/blob/main/toolkit/tools/imagecustomizer/container/imagecustomizer.Dockerfile#L30 - which is in `local`?